probe_scraper/probe_expiry_alert.py

```diff
@@ -20,21 +22,34 @@
 FROM_EMAIL = "telemetry-alerts@mozilla.com"
 DEFAULT_TO_EMAIL = "dev-telemetry-alerts@lists.mozilla.org"
 
+BUGZILLA_BUG_URL = "https://bugzilla.mozilla.org/rest/bug"
+BUGZILLA_USER_URL = "https://bugzilla.mozilla.org/rest/user"
+BUGZILLA_BUG_LINK_TEMPLATE = "https://bugzilla.mozilla.org/show_bug.cgi?id={bug_id}"
+
 BASE_URI = "https://hg.mozilla.org/mozilla-central/raw-file/tip/toolkit/components/telemetry/"
 HISTOGRAMS_FILE = "Histograms.json"
 SCALARS_FILE = "Scalars.yaml"
 EVENTS_FILE = "Events.yaml"
 
-EMAIL_BODY_FORMAT_STRING = """
-The following Firefox probes have either expired or will expire in the next major Firefox nightly release: version {next_version} [1].
+BUG_WHITEBOARD_TAG = "[probe-expiry-alert]"
+BUG_SUMMARY_TEMPLATE = "Remove or update probes expiring in Firefox {version}: {probe}"
+
+# Regex for version and probe names requires bug description to have a certain structure
+# This template should be modified with care
+BUG_DESCRIPTION_TEMPLATE = """
+The following Firefox probes will expire in the next major Firefox nightly release: version {version} [1].
 
+```
 {probes}
+```
+
+{notes}
 
 What to do about this:
 1. If one, some, or all of the metrics are no longer needed, please remove them from their definitions files (Histograms.json, Scalars.yaml, Events.yaml).
 2. If one, some, or all of the metrics are still required, please submit a Data Collection Review [2] and patch to extend their expiry.
 
-If you have any problems, please ask for help on the #fx-metrics Slack channel. We'll give you a hand.
+If you have any problems, please ask for help on the #fx-metrics Slack channel or the #telemetry Matrix room at https://chat.mozilla.org/#/room/#telemetry:mozilla.org. We'll give you a hand.
```

Comment:
Looking it over, let's also mention that it's easier to extend collection now, with the shorter form.